apps/cli/src/template.ts

```diff
@@ -22,3 +22,17 @@ export const download = async (
         providers: { cartesi: cartesiProvider },
     });
 };
+
+export const templates = () => {
+    return [
+        "cpp",
+        "cpp-low-level",
+        "go",
+        "javascript",
+        "lua",
+        "python",
+        "ruby",
+        "rust",
+        "typescript",
+    ];
+};
```

Comment:
let's keep constants upper case, and it doesn't need to be a function, remember to change this also on `apps/cli/src/commands/rollups/create.ts` for consistency.

```suggestion
export const TEMPLATES = [
        "cpp",
        "cpp-low-level",
        "go",
        "javascript",
        "lua",
        "python",
        "ruby",
        "rust",
        "typescript",
    ];
```